django_cloud_deploy/cli/prompt.py

```diff
@@ -924,6 +925,65 @@ def _validate(self, s: str):
                 ).format(s))
 
 
+class DjangoFilesystemPathCloudify(TemplatePrompt):
```

Comment:
Did you check StringTemplatePrompt Class? This looks like it would be the ideal case for it.

Reply:
I reimplemented this class with StringTemplatePrompt.